vmdb/spec/requests/api/tag_collections_spec.rb

```diff
@@ -7,21 +7,35 @@
 
   include Rack::Test::Methods
 
-  before(:each) do
-    init_api_spec_env
+  let(:zone)         { FactoryGirl.create(:zone, :name => "api_zone") }
+  let(:miq_server)   { FactoryGirl.create(:miq_server, :guid => miq_server_guid, :zone => zone) }
+  let(:ems)          { FactoryGirl.create(:ems_vmware, :zone => zone) }
+  let(:host)         { FactoryGirl.create(:host) }
+
+  let(:tag1)         { {:category => "department", :name => "finance", :path => "/managed/department/finance"} }
```

Comment:
This is making me wonder.
Is there some magic going on that is creating this tag?